tools/autograd/derivatives.yaml

```diff
@@ -277,6 +277,9 @@
 
 - name: is_same_size  # fallthrough
 
+- name: is_signed(Tensor self)
```

Comment:
this doesn't actually work right now because we don't generate a VariableType in this case; that should be solvable, although maybe in a different PR.  Also, we should rename fallthrough to something else -- it's confusing because these are actually doing composition.  Maybe "unwrap" is the correct name.